src/components/pacbio/PacbioLibraryCreate.vue

```diff
@@ -22,10 +22,20 @@
           {{ selectedSample.sample_name }} ({{ selectedSample.source_identifier }})
         </traction-form-group>
 
-        <traction-form-group id="tag-select-input" label="Tag:" label-for="tag-input">
+        <traction-form-group id="tag-set-select-input" label-for="tag-set-input" label="Tag:">
+          <traction-select
+            id="tag-set-input"
+            v-model="selectedTagSetId"
+            data-type="tag-set-list"
+            :options="tagSetOptions"
+            @input="resetSelectedTagId"
+          ></traction-select>
+        </traction-form-group>
+
+        <traction-form-group id="tag-select-input" label-for="tag-input">
           <traction-select
             id="tag-input"
-            v-model="library.tag.group_id"
+            v-model="selectedTagId"
```

Comment:
why did this need changing?

Reply:
It is for changing the logic in createLibraryInTraction. Instead of querying again in there to find the tag.id from tag.group_id, I directly used the Id of the tag.

Reply:
I have changed this line back to library.tag.id . Note that we are not using group_id as it is not necessary. The previous code had another step to find id from group_id in createLibraryInTraction. We don't need that anymore as the id is available.